src/components/AboutContainer/AboutContainer.test.js

```diff
@@ -14,23 +14,23 @@ describe('<AboutContainer />', () => {
 
   const optionLabels = ['What We Do', 'Who We Are', 'Programs', 'Get Involved'];
 
-  it('should display banner image', () => {
+  it.skip('should display banner image', () => {
```

Comment:
why are we skipping the test; will approve this PR because Krizia wants this deployed, but let's fix this later